task_processing/plugins/kubernetes/utils.py

```diff
@@ -126,17 +57,21 @@ def get_kubernetes_env_vars(
     env_vars = [
         V1EnvVar(name=key, value=value) for key, value
         in environment.items()
-        if not is_secret_env_var(value)
     ]
 
     secret_env_vars = [
-        get_secret_kubernetes_env_var(
-            key=key, value=value, task_name=task_name, namespace=namespace
+        V1EnvVar(name=key, value_from=V1EnvVarSource(
+            secret_key_ref=V1SecretKeySelector(
+                name=value["secret"],
+                key=value["key"],
+                optional=False,
+            ),
+        ),
         )
```

Comment:
super-nit: can we update the formatting a tiny bit here? we should probably add `black` to this repo at some point tbh though :p
```suggestion
        V1EnvVar(
            name=key,
            value_from=V1EnvVarSource(
                secret_key_ref=V1SecretKeySelector(
                    name=value["secret"],
                    key=value["key"],
                    optional=False,
                ),
            ),
        )
```

Reply:
Ah, right.  I do think *something* in our precommits likes to change this out from under me, because this annoys me as well (why would closing params for nested calls end at the same indentation?!).  I'll try to see if I can work around it.

Reply:
worst case, we can merge this as-is and just get around to running black on this repo next week and stop caring about this :p

Reply:
Turns out this one was my fault, since `secret_key_ref` was not indented correctly to begin with.  precommits did not argue argue with me this time!